app/src/organisms/ProtocolSetup/RunSetupCard/ModuleSetup/UnMatchedModuleWarning.tsx

```diff
@@ -26,48 +20,50 @@ export const UnMatchedModuleWarning = (
   props: UnMatchedModuleWarningProps
 ): JSX.Element | null => {
   const { t } = useTranslation('protocol_setup')
-  const [showModulesMismatch, setShowModulesMismatch] = React.useState<boolean>(
-    true
-  )
-  const isVisible = showModulesMismatch && props.isAnyModuleUnnecessary
+  const isVisible = props.isAnyModuleUnnecessary
   if (!isVisible) return null
 
   return (
-    <React.Fragment>
+    <Flex
+      marginTop={SPACING.spacing4}
+      flexDirection={DIRECTION_COLUMN}
+      backgroundColor={COLORS.background}
+      padding={SPACING.spacing5}
+    >
       <Flex
-        marginY={SPACING_3}
-        backgroundColor={COLOR_WARNING_LIGHT}
-        color={C_DARK_GRAY}
-        id={'ModuleMismatch'}
+        flexDirection={DIRECTION_ROW}
+        justifyContent={JUSTIFY_SPACE_BETWEEN}
       >
-        <Flex flexDirection={DIRECTION_COLUMN} flex={'auto'} margin={SPACING_3}>
-          <Flex justifyContent={JUSTIFY_SPACE_BETWEEN}>
-            <Flex>
-              <Box size={SIZE_2} paddingY={SPACING_1} paddingRight={SPACING_2}>
-                <Icon name="alert-circle" color={COLOR_WARNING} />
-              </Box>
-              <Text
-                as="h4"
-                marginY={SPACING_2}
-                id={`ModuleMismatch_title`}
-                color={COLOR_WARNING_DARK}
-              >
-                {t('module_mismatch_title')}
-              </Text>
-            </Flex>
-            <Btn
-              size={SIZE_2}
-              onClick={() => setShowModulesMismatch(false)}
-              aria-label="close"
-            >
-              <Icon name={'close'} color={COLOR_WARNING} />
-            </Btn>
-          </Flex>
-          <Text paddingTop={SPACING_1} fontSize={'0.7rem'}>
-            {t('module_mismatch_body')}
-          </Text>
+        <Flex>
+          <Icon
+            size={SPACING.spacing6}
+            color={COLORS.darkGreyEnabled}
+            name="information"
+            paddingRight={SPACING.spacing3}
+            paddingBottom={'0.625rem'}
+            aria-label="information_icon"
+          />
+          <StyledText
+            fontSize={TYPOGRAPHY.fontSizeP}
+            fontWeight={TYPOGRAPHY.fontWeightSemiBold}
+            data-testid={`UnMatchedModuleWarning_title`}
+            color={COLORS.darkBlack}
```

Comment:
The default text color is `darkBlack`, so we don't need that.
Let's use `as="p"` for consistency. Actually you use that for L57-L66
```suggestion
          <StyledText
            as="p"
            fontWeight={TYPOGRAPHY.fontWeightSemiBold}
            data-testid={`UnMatchedModuleWarning_title`}
```